bin/node/cli/src/command.rs

```diff
@@ -95,6 +95,22 @@ pub fn run() -> Result<()> {
 				You can enable it with `--features runtime-benchmarks`."
 					.into())
 			},
+		Some(Subcommand::BenchmarkStorage(cmd)) => {
+			if cfg!(feature = "runtime-benchmarks") {
+				return Err("Benchmarking wasn't enabled when building the node. \
+				You can enable it with `--features runtime-benchmarks`."
```

Comment:
it would be nice if this was a subcommand of benchmark otherwise everyone would need to update their CLI to include this

Reply:
The `new_chain_ops` function that I call here is located in `node/service`. It is not possible to call it from Substrate.  
This sets up a client, I think setting up a client is the responsibility of the CLI.

Reply:
Thinking again: we could rename it to `benchmark-client` and add sub-commands to that since I think that the Ext and Block benchmarks will need a client as well.

Reply:
can be done in the future, np